### PR TITLE
changes needed for cert manager ~> 0.12

### PR DESCRIPTION
**Pull request by jbianquetti-nami**
This pull request should NOT be merged until we already have our clusters running cert-manager 0.12.X

### CODE REVIEW

File: bitnami.libsonnet [hunk 1]
```diff
@@ -71,14 +71,14 @@ local perCloudSvcSpec(cloud) = (
...
+        "cert-manager.io/cluster-issuer": "letsencrypt-prod-dns",
+        "cert-manager.io/acme-challenge-type": "dns01",
+        "cert-manager.io/acme-dns01-provider": "default",
       },
     },
     cm_http_metadata:: {
       annotations+: {
-        "certmanager.k8s.io/cluster-issuer": "letsencrypt-prod-http",
+        "cert-manager.io/cluster-issuer": "letsencrypt-prod-http",
       },
```
Comment (aruiz14) [inline bitnami.libsonnet]: Cannot these annotations co-exist so it's safe to land for now? Once the migration happens, then setting these values to `null` should effectively delete the annotations from the resources (just dropping them will usually just leave them unmanaged but still present).

Comment (jjo) [reply bitnami.libsonnet]: big +1

File: bitnami.libsonnet [hunk 2]
```diff
@@ -71,14 +71,14 @@ local perCloudSvcSpec(cloud) = (
     },
     cm_dns_metadata:: {
       annotations+: {
-        "certmanager.k8s.io/cluster-issuer": "letsencrypt-prod-dns",
-        "certmanager.k8s.io/acme-challenge-type": "dns01",
-        "certmanager.k8s.io/acme-dns01-provider": "default",
+        "cert-manager.io/cluster-issuer": "letsencrypt-prod-dns",
```
Comment (jjo) [inline bitnami.libsonnet]: As I understand, only `cert-manager.io/cluster-issuer` new annotation needs to be present, as the old annotation selectors (acme-challenge-type and acme-dns01-provider)  are deprecated, as the cluster-issuer no has a specific `selector` field for that (~"server-side").

Comment (jbianquetti-nami) [reply bitnami.libsonnet]: I've got these changes on my local branch: pushing it to this PR

File: bitnami.libsonnet (hunk 1 above)
Comment (jbianquetti-nami) [reply bitnami.libsonnet]: Well, they are coexisting right now at some clusters, so we can land it now.

**Comment by jbianquetti-nami** [comment]
@aruiz14 @jjo PTAL

File: bitnami.libsonnet (hunk 1 above)
Comment (aruiz14) [reply bitnami.libsonnet]: BTW I was wrong, since [kubecfg 0.12.0](https://github.com/bitnami/kubecfg/releases/tag/v0.12.0 )we don't need to set to null anymore. Just removing the label/annotation will do it:
> TL;DR: Previously, kubecfg treated "unspecified" fields as "don't care". Now it interprets "unspecified" fields that were previously specified as "set to default". So now you can remove an annotation/label from your jsonnet, and kubecfg update will remove it from the server.
